content/pl_docker/docker_course_dev.md

```diff
@@ -173,3 +173,49 @@ When developing locally on PrairieLearn, you **must** click "Load from Disk" eve
 
 That's it!
 You're ready to develop questions for the OPB!
+
+# Setting up local autograder container
+For development of questions that require an autograder, you will need to set up a local autograder container as well. 
+
+## Important! (Windows Only)
+If you are using Windows, the following commands will need to be executed in Windows Subsystem for Linux (WSL). You can do this two ways:
+1. Open a command prompt and type `wsl` to open a WSL terminal
+1. Look for the Windows Subsystem for Linux application in the Start Menu
+<img src="pl_images/wsl_icon_windows.png">
+
+## Step 1: Create a local directory for the autograder
+
+To run PrairieLearn locally with external grader and workspace support, create an empty directory to use to share job data between containers. This directory can live anywhere, but needs to be created first and referenced in the docker launch command. This directory only needs to be created once.
+```bash
+mkdir "$HOME/pl_ag_jobs"
+```
```

Comment:
add a line break between code-fence and preceding text for readability (apply everywhere)